src/evaluate.cpp

```diff
@@ -1083,13 +1083,13 @@ namespace {
 Value Eval::evaluate(const Position& pos) {
 
   Value v;
-  bool useClassical = false;
+  bool useClassical = (pos.this_thread()->depth > 9 || pos.count<ALL_PIECES>() > 7) &&
+                      abs(eg_value(pos.psq_score())) * 5 > (856 + pos.non_pawn_material() / 64) * (10 + pos.rule50_count());
 
   // Deciding between classical and NNUE eval (~10 Elo): for high PSQ imbalance we use classical,
   // but we switch to NNUE during long shuffling or with high material on the board.
   if (  !useNNUE
-      || ((pos.this_thread()->depth > 9 || pos.count<ALL_PIECES>() > 7) &&
-          abs(eg_value(pos.psq_score())) * 5 > (856 + pos.non_pawn_material() / 64) * (10 + pos.rule50_count())))
+      || useClassical)
```

Comment:
I like this cleanup, makes code easier to read, but the comments describing the code should move with the code.